src/Raven.Server/Documents/Queries/Graph/IQueryPlan.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using static Raven.Server.Documents.Queries.GraphQueryRunner;
+
+namespace Raven.Server.Documents.Queries.Graph
+{
+    public interface IQueryPlan 
```

Comment:
Why do we need the `QueryPlan` to be an interface?

Reply:
Nope